src/main/java/io/supertokens/utils/Utils.java

```diff
@@ -432,6 +434,12 @@ public static JsonElement toJsonTreeWithNulls(Object src) {
         return new GsonBuilder().serializeNulls().create().toJsonTree(src);
     }
 
+
+    public static boolean isAccountLinkingEnabled(Main main, AppIdentifier appIdentifier) throws StorageQueryException, TenantOrAppNotFoundException {
+        return Arrays.stream(FeatureFlag.getInstance(main, appIdentifier).getEnabledFeatures())
+                .anyMatch(t -> t == EE_FEATURES.ACCOUNT_LINKING || t == EE_FEATURES.MFA);
+    }
```

Comment:
not sure why this change appears here

Reply:
Because this was moved from `AuthRecipe.java `to the `Utils.java` as it was used in `BulkImportUserUtils` too, but with code duplication. I removed the code duplication and extracted this to a place where both participants can reach and doesn't go against the dependencies between the code layers.